package.json

```diff
@@ -33,7 +33,7 @@
     "k3d-setup": "k3d cluster delete pepr-dev && k3d cluster create pepr-dev --k3s-arg '--debug@server:0'"
   },
   "dependencies": {
-    "pepr": "0.27.0"
+    "pepr": "0.28.3"
```

Comment:
Commenting here for visibility - not sure if we want to wait on the [Ironbank](https://repo1.dso.mil/dsop/opensource/defenseunicorns/pepr/controller) update here...but that image is technically still pinned at 0.27.0 - https://github.com/defenseunicorns/uds-core/blob/making-better-controllers/tasks/create.yaml#L7

Reply:
Yeah I was hoping to get this wrapped up this week, but pushed on IB side, so I guess we'll see what happens with them tomorrow.

Reply:
0.28.3 is publishing in IB now - prioritizing the rest of this review to get it merged and released today.